From: Kalev Lember <klember@redhat.com>
Date: Sat, 15 Sep 2018 15:27:44 +0000
Subject: color: Fix fallout from the GtkTemplate port

Correct a typoed variable name so that calibration works again, instead
of crashing.

GLib-GObject-WARNING **: 15:52:50.561: invalid cast from 'GtkBox' to 'GtkEntry'

(cherry picked from commit edc52c170d3d58c4d767f8e8b1d6cbb9381352d4)

Origin: upstream, 3.30.1
---
 panels/color/cc-color-panel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/color/cc-color-panel.c b/panels/color/cc-color-panel.c
index 2a30f35..ee26cfe 100644
--- a/panels/color/cc-color-panel.c
+++ b/panels/color/cc-color-panel.c
@@ -574,7 +574,7 @@ gcm_prefs_title_entry_changed_cb (GtkWidget *widget,
   const gchar *value;
 
   assistant = GTK_ASSISTANT (prefs->assistant_calib);
-  value = gtk_entry_get_text (GTK_ENTRY (prefs->box_calib_title));
+  value = gtk_entry_get_text (GTK_ENTRY (prefs->entry_calib_title));
   cc_color_calibrate_set_title (prefs->calibrate, value);
   gtk_assistant_set_page_complete (assistant, prefs->box_calib_title, value[0] != '\0');
 }
